scissors/scissors_game/main.c

```diff
@@ -3,31 +3,42 @@
 #include <stdlib.h>
 #include <time.h>
 
-//"-1" - draw
-// "0" - fail
-// "1" - win
+typedef enum {
+	eDRAW = -1,
+	eLOST,
+	eWINN
+} decision_t;
+
+/*Enum of Unknown, Rock, Paper and Scissors choise*/
+typedef enum {
+	eU = -1,
+	eR,
+	eP,
+	eS
+
+} element_t;
```

Comment:
Please use meaningful enum names instead if comments for them.
Also drop empty string in enum.

This patch should be squashed into the previous one.